zdm/survey.py

```diff
@@ -713,44 +970,160 @@ def load_survey(survey_name:str, state:parameters.State, dmvals:np.ndarray,
     """
     print(f"Loading survey: {survey_name}")
     if sdir is None:
-        sdir = os.path.join(resource_filename('zdm', 'data'), 'Surveys')
+        sdir = os.path.join(
+            resource_filename('zdm', 'data'), 'Surveys')
+        if original:
+            sdir = os.path.join(sdir, 'Original')
 
     # Hard code real surveys
```

Comment:
Hi @profxj I've been thinking about a few survey parameters, in particular nbins for beams, and how to parameterise efficiency. I *think* I've concluded that nbins needs to either go in the survey file itself, or at least we should allow for a default value there that can be over-written explicitly. In any case, I'd like to chat about this at some point, and need to think a bit more about what CHIME needs. This is of course fine for now, but it might need to be changed

Reply:
`nbins` now part of the data model.

We may want an analysis section for such parameters